configs/_base_/models/hv_pointpillars_secfpn_kitti.py

```diff
@@ -14,7 +14,8 @@
         feat_channels=[64],
         with_distance=False,
         voxel_size=voxel_size,
-        point_cloud_range=[0, -39.68, -3, 69.12, 39.68, 1]),
+        point_cloud_range=[0, -39.68, -3, 69.12, 39.68, 1],
+        legacy=False),
```

Comment:
Would better to keep the current Default setting because the current one is the "correct" version even though it can get a little worse performance

Reply:
We can add a note or some comments to indicate the effect of this default setting to the detection performance

Reply:
Good suggestion, In fact, turning legacy to `False` only causes minor performance drop.